### ISSUE TITLE
On using the washboard some things have to be washed several times.

### ISSUE THREAD

**Issue by Androwski3000** [CleverRaven/Cataclysm-DDA#24907]
If the clothes are meant to be so dirty that they need washing multiple times, there should be a grade of filth be introduced.

Like:
-Ultra filthy,
-stands in the corner alone by itself,
-yak,
-get me out of here,
-get this out of here.

This would make things more clear.

Or just make it so that it is clean after one wash, as some things in the game kept simple. Like meat is meat. regardless from which creature it is, unless it was a zombie. The same with fish. Fish filet is fish filet, no matter what kind of fish it was before.

**Comment by Night-Pryanik** [comment]
Please provide us a reproducible case of when you needed to wash clothes twice.

**Comment by alanbrady** [comment]
I have encountered this before.  Haven't seen it recently.  I assumed it was on purpose.  I can't recall the exact scenario, but I have seen it.

**Comment by Night-Pryanik** [comment]
This is definitely not intended. If it's reproducible, it's a bug.

**Comment by Androwski3000** [comment]
@Night-Pryanik : Right know, I don´t have much things ingame which need washing. But I´ll try to reproduce that and will provide a savegame also.
I had this issue to wash items several times at least, when I washed like 4 items at the same time via the washboard menu. I can´t tell exactly if I had this issue with only one single item.

I need firefigther items soon for kevlar. And I cleared 3 firestations where there is still firefighter items from zombies lying around.

**Comment by Duhuh123** [comment]
This has happened in my game too.

**Comment by Androwski3000** [comment]
I picked up  all firefighter gear from 2 firefighter stations to get kevlar. It was all filthy.
On washing 99% was the firefighter gear. I think I had the issue again once maybe 10 washings. Overall 500-100 items have been washed.
On that occurance I forgot to make a screenshot before. :-/ At the washings I made a screenshot, nothinh happened.
I notice the issue as follows:

I activate the washboard via "a" and then select all items for washing. Only items from the player inventory are taken.
I wash. The log states it´s text like: All items are washed.
Then I discover filthy items in my inventory.

Maybe it has to do with multiple items on one stack. Where only one item of the stack is washed.

**Comment by kevingranade** [comment]
The stack guess is possible, and I also wonder if it could be caused by the
code moving around some items and losing track of them.

If it's reaching capacity and only washing part of a stack, that's working
as intended.

**Comment by Androwski3000** [comment]
I could track one of theese non washings via screenshot. 
Before I left the shelter to go looting, I checked if I carry unneeded things. The windbreaker I washed some minutes before, was still filthy.
I have the success report about washing only in the log "P". I forgot the before screenie in the wash menu.

![no wash1](https://user-images.githubusercontent.com/42423698/44473257-2f7a8600-a630-11e8-8fcd-8422da0f8d89.jpg)

![no wash2](https://user-images.githubusercontent.com/42423698/44473305-50db7200-a630-11e8-9007-540ceed0fb6c.jpg)
